lib/tokenlist.cpp

```diff
@@ -2051,14 +2051,13 @@ bool TokenList::isKeyword(const std::string &str) const
         if (cpp_types.find(str) != cpp_types.end())
             return false;
 
-        // TODO: properly apply configured standard
-        if (!mSettings || mSettings->standards.cpp >= Standards::CPP20) {
-            static const auto& cpp20_keywords = Keywords::getAll(Standards::cppstd_t::CPP20);
-            return cpp20_keywords.find(str) != cpp20_keywords.end();
+        if (mSettings) {
+            const auto &cpp_keywords = Keywords::getAll(mSettings->standards.cpp);
+            return cpp_keywords.find(str) != cpp_keywords.end();
```

Comment:
imho `cpp_keywords.count(str)` would be less code. The count and your code does the same thing. And you can't mistype count i.e. `a.find(str) != b.end()`. Of course, when c++20 will become available it will be syntactically nicer to replace count with contains.